javascript/src/main/components/Courses/CourseDetail.js

```diff
@@ -0,0 +1,13 @@
+  
```

Comment:
Unnecessary extra line? (This also seems to be present at the beginning of many of the other files)

Reply:
I will remove this line in the next commit